### PR TITLE
Fixes #37378 - Update TableWrapper to comply with changes in SelectAllCheckbox

### PR DESCRIPTION
**Pull request by ofedoren**
#### What are the changes introduced in this pull request?

1. commit: updates for TableWrapper
2. commit: make workflow to test it with https://github.com/theforeman/foreman/pull/10113

#### Considerations taken when implementing this change?

https://github.com/theforeman/foreman/pull/10113#issuecomment-2061541628

#### What are the testing steps for this pull request?

Run the tests, hope for the best.

P.S. After it's green, I'll remove the second commit.

### CODE REVIEW

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -25,3 +25,4 @@ jobs:
       plugin: katello
       postgresql_container: ghcr.io/theforeman/postgresql-evr
       test_existing_database: false
+      foreman_version: refs/pull/10113/head
```
Comment (jeremylenz) [inline .github/workflows/ruby.yml:28]: TIL! :exploding_head:

**Comment by jeremylenz** [comment]
🍏 

let's get that 2nd commit outta here and then I can merge both PR's!

**Comment by ofedoren** [comment]
done :)

**Review by jeremylenz — APPROVED** [review]
Tests pass when tested with Foreman PR

thanks @ofedoren !

**Pull request merged**